src/main/java/com/icthh/xm/ms/dashboard/service/bulk/impl/AtomicBulkDashboardServiceImpl.java

```diff
@@ -0,0 +1,84 @@
+package com.icthh.xm.ms.dashboard.service.bulk.impl;
+
+import static java.util.stream.Collectors.toList;
+
+import com.icthh.xm.commons.exceptions.BusinessException;
+import com.icthh.xm.ms.dashboard.domain.Dashboard;
+import com.icthh.xm.ms.dashboard.mapper.DashboardMapper;
+import com.icthh.xm.ms.dashboard.repository.DashboardRepository;
+import com.icthh.xm.ms.dashboard.service.bulk.AtomicBulkDashboardService;
+import com.icthh.xm.ms.dashboard.service.dto.BulkDashboardResult;
+import com.icthh.xm.ms.dashboard.service.dto.DashboardDto;
+import java.util.Collection;
+import javax.transaction.Transactional;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+@Service
+@RequiredArgsConstructor
+public class AtomicBulkDashboardServiceImpl implements AtomicBulkDashboardService {
+
+    private final DashboardMapper dashboardMapper;
+    private final DashboardRepository dashboardRepository;
+
+    @Override
+    public BulkDashboardResult create(Collection<DashboardDto> dashboardItems) {
+        try {
+            Collection<Dashboard> dashboardEntities = dashboardItems.stream()
+                .map(dashboardMapper::toEntity)
+                .collect(toList());
+
+            save(dashboardEntities);
+            return new BulkDashboardResult();
+        } catch (Exception ex) {
+            throw new BusinessException("Could not perform bulk create : " + ex.getMessage());
```

Comment:
better without try catch

Reply:
Hi @sergeysenja1992 it would be great if you could clarify your above comment with objectives if so

Reply:
1) No reason to hide real exception and replace error code. For this reason present ExceptionTranslator. 
2) In this case hidden information about stacktrace, and it's big problem for problem identification.
3) BusinessException - it's mean BUSINESS, we will throw it by some BUSINESS logic rules.

The same in other cases in this pull request.

Reply:
Thanks for a detailed explanation. I already throws where it is needed business exception and removed that from other places, thanks!